lib/box-client.js

```diff
@@ -643,7 +643,7 @@ BoxClient.prototype.wrapWithDefaultHandler = function(method) {
 					return response.body;
 				}
 
-				throw errors.buildResponseError(response);
+				throw errors.buildResponseError(response, response.body.code);
```

Comment:
It might actually be better to have parity with the default handler for callbacks above and do `throw errors.buildUnexpectedResponseError(response)`.  If you don't mind making that change, I'm happy to merge this after!